.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: java
 jdk:
   - oraclejdk8
-sudo: required
 
-jobs:
-  include:
-    - stage: build
-      script: mvn clean package
-      if: type = push AND branch != master
-    - stage: test
-      script:
-        - chmod +x install.sh && ./install.sh > /dev/null
-        - set -o pipefail
-        - mvn -P integration-test-azure clean cobertura:cobertura-integration-test | grep -v "DEBUG"
-        - bash <(curl -s https://codecov.io/bash)
-      if: type = pull_request OR (type = push AND branch = master)
+before_script:
+  - chmod +x ./src/libs/integration-test-setup.sh
+  - ./src/libs/integration-test-setup.sh > src/test/resources/application.properties
+  - cat src/test/resources/application.properties
```

Comment:
Wonder is it better not to cat the properties? Without revealing the key and secret.

Reply:
I put this cat here just FYI, for make sure the secret key is protected correctly. We can remove this in .travis.yml. But I recommend to keep it.